packages/stockflux-launcher/src/titlebar/Titlebar.js

```diff
@@ -0,0 +1,68 @@
+import cx from 'classnames';
+import React, { useCallback } from 'react';
+import { OpenfinApiHelpers } from 'stockflux-core';
+import Components from 'stockflux-components';
+import './Titlebar.css';
+
+export default ({ dockedTo }) => {
+  const onMinimizeClick = useCallback(async () => {
+    const currentWindow = await OpenfinApiHelpers.getCurrentWindow();
+    currentWindow.minimize();
+  }, []);
+
+  const onCloseClick = useCallback(async () => {
+    const currentWindow = await OpenfinApiHelpers.getCurrentWindow();
+    currentWindow.close();
+  }, []);
+
+  const dockedHorizontal = dockedTo === 'top' || dockedTo === 'none';
+
+  return (
+    <div className="titlebar">
+      {dockedHorizontal && (
+        <div
+          className={cx(
+            'column column-left',
+            dockedTo === 'top' || dockedTo === 'none' ? 'add-padding-left' : ''
```

Comment:
Should we not use `ScreenEdge` here? Also could be simplified by `[ScreenEdge.None, ScreenEdge.Top].includes(dockedTo)`;

Reply:
It would definitely be worth having this as a helper function.